packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1263,10 +1261,6 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         this.maxOpsSinceLastSummary = this.getMaxOpsSinceLastSummary();
         this.initialSummarizerDelayMs = this.getInitialSummarizerDelayMs();
 
-        this._aliasingEnabled =
-            (this.mc.config.getBoolean(useDataStoreAliasingKey) ?? false) ||
-            (runtimeOptions.useDataStoreAliasing ?? false);
-
```

Comment:
Since behavior could be driven by the `Fluid.ContainerRuntime.UseDataStoreAliasing` option, I think the breaking changes notes should mention that it is not supported anymore. And I found another reference to that constant in `packages/test/test-utils/src/TestConfig.ts`, shouldn't impact anything but it'd be cleaner to remove it too?

Reply:
Good call. Will do